src/main/scala/me/rexim/morganey/ReplAutocompletion.scala

```diff
@@ -72,13 +73,18 @@ class ReplAutocompletion(globalContext: () => InterpreterContext) extends Comple
         .map { case (root, f) => (root, stripExtensionIfModuleFile(f)) }
         .filter { case (root, f) => fileNameFilter(f.getName) }
         .map(relativize)
+        .map(fixRelativePathForWindows)
         .map(relativeFileToLoadPath)
 
     def relativize(baseAndFile: (File, File)): String = {
       val (base, file) = baseAndFile
       base.toURI().relativize(file.toURI()).getPath()
     }
 
+    def fixRelativePathForWindows(path: String): String =
+      if (runningOnWindows) path.replace('/', File.separatorChar)
```

Comment:
Why does it have to be a windows specific thing? I mean let's get rid of this hacky condition and always replace `/` with `File.separateChar`. It will always work on any operating system. On Unix it will replace `/` with `/`. On Windows it will replace `\` with `/`. On anything else it will replace something else with `/`. And such approach actually hits more operating system and requires less code. What do you think?